workflows/test.yml

```diff
@@ -31,7 +31,7 @@ jobs:
 
       - name: Set NATS Server Version
         run: echo '::set-env name=NATS_VERSION::v2.1.7'
-      - name: Get nats-server
+      - name: Get nats-servers
```

Comment:
Is that a rename-all kind of things? Not sure why you would have change nats-server to nats-servers here :-)

Reply:
weird my local version doesn't have it. - I will fix the title on merge.